bin/workflows/pycbc_make_uberbank_workflow

```diff
@@ -219,67 +233,78 @@ geom_dax_job = dax_job
 sbank_exe = SbankDaxGenerator(workflow.cp, 'sbank_workflow',
                               ifos=workflow.ifos, out_dir='daxes')
 
-sbank_out_file_bbh = wf.File(sbank_exe.ifo_list, sbank_exe.name,
-                             workflow.analysis_time, extension='.h5',
-                             store_file=True, directory=sbank_exe.out_dir,
-                             tags=['sbank_bbh'], use_tmp_subdirs=False)
-
 config_path = os.path.abspath('daxes' + '/' + 'sbank_workflow.ini')
 workflow.cp.write(open(config_path, 'w'))
 config_file = wf.File.from_path(config_path)
 
-sbank_node = sbank_exe.create_node(workflow.analysis_time, config_file,
-                                   os.path.abspath('sbank_bbh'),
-                                   sbank_out_file_bbh,
-                                   workflow_name='sbank_bbh',
-                                   sub_wf_tags=['bbh'])
-
-workflow += sbank_node
-
-dax_file = sbank_node.output_files[0]
-map_file = sbank_node.output_files[1]
-dax_job = dax.DAX(dax_file)
-dax_job.addArguments('--basename %s' % \
-                     os.path.splitext(os.path.basename(dax_file.name))[0])
-wf.Workflow.set_job_properties(dax_job, map_file)
-
-workflow._adag.addJob(dax_job)
-dep = dax.Dependency(parent=sbank_node._dax_node, child=dax_job)
-workflow._adag.addDependency(dep)
-sbank_bbh_dax_job = dax_job
+# This first round of sbank is used for BBHs for the standard uberbank;
+# a PyGRB uberbank covers only potentially EM-bright systems and therfore
+# skips this step.
+if workflow.cp.has_section('sbank-coarse'):
```

Comment:
Following the config files guidelines, the existence (or not) of an executable section should not be used to determine workflow details.

Can you instead put some switch in *any* section beginning `workflow`. E.g.
```
[workflow-bank_structure]
skip-coarse-sbank=
```
With the default behaviour being to run this.